opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -486,6 +510,9 @@ public void onSyncInProgress(FetchStatus fetchStatus) {
         } else if (fetchStatus.equals(FetchStatus.fetched)
                 || fetchStatus.equals(FetchStatus.nothingFetched)) {
             Snackbar.make(rootView, org.smartregister.R.string.sync_complete, Snackbar.LENGTH_LONG).show();
+            if (getInterventionLabel() == R.string.focus_investigation) {
+                addCaseIndexBoundary(mMapboxMap, getContext());
```

Comment:
This should be removed

Reply:
Doesn't that only happen if new structures were fetched?

Reply:
No, also when structures are changed e.g due to spray forms updating the business status

Reply:
Done.